takeshape/penny
# Fix price selection, update fixtures

### Test Plan (Steps to test):

1. Go to the Awesome T-shirt on the preview site
1. Change your price selection
1. Change your variant, your price selection should stick
